### PR TITLE
Remove css error modules

### PR DESCRIPTION
**Pull request by kmck**
Removes the weird error modules thing that this was doing in favor of just returning the error or throwing. Tests updated to reflect this.

### CODE REVIEW

File: test/sassr.test.js [hunk 1]
```diff
@@ -18,28 +18,34 @@ var EXPECTED_ERROR = '"property \\"We\\" must be followed by a \':\'"';
...
-        bad: fs.readFileSync(scssPaths.bad).toString()
+        bad: fs.readFileSync(scssPaths.bad).toString(),
     };
 
     var txtPath = require.resolve('./fixtures/not-css.txt');
 
     function assertCSSText(source, cssText, callback) {
+        callback = callback || function() {};
         var module = testUtils.loadAsModule(source);
         assert.equal(module.getCSSText(), cssText);
```
Comment (mysterycommand) [inline test/sassr.test.js]: If `assert.equal` is in the `try` (right below) does it still need to be here?

**Comment by mysterycommand** [comment]
Apart from the one comment, looks good to me! :+1:

File: test/sassr.test.js (hunk 1 above)
Comment (kmck) [inline test/sassr.test.js]: ah, good call. missed that line when i was cherry picking from another branch

**Comment by heylookltsme** [comment]
LGTM

**Pull request merged**